app/src/main/AndroidManifest.xml

```diff
@@ -9,13 +9,13 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.MyCalculator">
-        <activity android:name=".MainActivity">
+        <activity
+            android:name=".MainActivity"
+            android:configChanges="orientation|screenSize">
```

Comment:
why do you add this line?

Reply:
I used that property since it allowed me that when rotating the phone, the values ​​of the operation would not be lost.

Reply:
yes its a valid use of the changes of the screen https://developer.android.com/guide/topics/resources/runtime-changes#:~:text=Possible%20values%20are%20listed%20in,when%20the%20screen%20orientation%20changes.